Cargo.toml

```diff
@@ -11,7 +11,7 @@ name = "bootupd"
 path = "src/bootupd.rs"
 
 [[bin]]
-name = "bootupd"
+name = "bootupctl"
```

Comment:
If you ever get back to this, I'd much prefer keeping the previous binary name and hardlink `bootupctl` later to it.

Reply:
Yeah I did it that way in a previous version of this PR, it worked fine but then I realized...what's the point?  The fact that it's a daemon is ideally mostly an implementation detail.

This PR also changes the CLI to split the "end user interface" off from the "OS builder" side so it's an incompatible change anyways.  I could keep the previous version but eh.

(i.e. we should think of it more like "bootupd-ater" than "bootupd-aemon")